diagnostics/inspection.py

```diff
@@ -24,46 +24,48 @@
 from . import utils
 
 
+sys.setrecursionlimit(5000)
+
+
 def inspect_samples(y0: Tensor,
                     ts: Vector,
                     dt: Scalar,
                     sde: BaseSDE,
                     bm: BaseBrownian,
                     img_dir: str,
                     methods: Tuple[str, ...],
-                    options: Optional[Tuple] = None,
+                    options: Tuple,
```

Comment:
Is there a particular reason to remove the default `None` for `options` and `labels`?